Sources/ApodiniDatabase/Handlers/Read.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Fluent
+import Apodini
+
+/// A Handler that returns an array of `DatabaseModel` which all conform to the query parameters of the request. It accepts all parameters that represent properties of the specified `DatabaseModel`.
```

Comment:
Sorry for being that picky here, but since there is an open change-request anyways, please add line-breaks after ~150 characters on block-comments.